contrib/win32/win32compat/misc.c

```diff
@@ -257,54 +264,96 @@ int w32_ioctl(int d, int request, ...) {
         }
 }
 
-int 
-spawn_child(char* cmd, int in, int out, int err, DWORD flags) {
+
+int spawn_child_vp(char * path, char ** argv, int in, int out, int err, DWORD flags) {
```

Comment:
I named, by analogy with **execv** function: https://www.freebsd.org/cgi/man.cgi?query=execv
In the future will need replace the call **spawn_child** to call **spawn_child_vp**:
1) https://github.com/PowerShell/openssh-portable/blob/c6b57c018c3ea532854c404c9be194bce6cb51e2/scp.c#L315
2) https://github.com/PowerShell/openssh-portable/blob/c6b57c018c3ea532854c404c9be194bce6cb51e2/sftp.c#L2351